sample/src/main/java/no/finn/android/finnflow/sample/BaseActivity.java

```diff
@@ -10,4 +12,9 @@
     protected Screen createDefaultScreen() {
         return new MasterScreen();
     }
+
+    @Override
+    protected StateParceler getParcer() {
```

Comment:
You create a new parcer here every time you trigger it?

Maybe instead have a getParcer in the main code, and rename this to createParcer. Then the "base" code can contain logic for keeping the reference and whoever implements this doesn't need to worry about it.